### DISCUSSION TITLE
How to Change Extruder Motor Rotation Direction?

### DISCUSSION THREAD

**Discussion by 1RonRock** [luc-github/Repetier-Firmware-0.92#377] [Q&A]
I have a Davinci 1.0a with the Repetier firmware loaded. I have a E3D-V6 hotend and now would like to install the E3D Titan Extruder. That moves the motor from rear of the carriage to front. That in effect requires the motor to reverse direction.

I think that I have found the correct setting that I need to change in my Configuration h file.

#endif
#if DAVINCI == 1 || DAVINCI == 4
#define EXT0_INVERSE false
#endif

Do I change,

#define EXT0_INVERSE false

to

#define EXT0_INVERSE true

Will that change direction of the extruder motor?

**Comment by luc-github** [comment]
yes that is the correct setting

**Reply by 1RonRock** [reply]
Thank you. 

Finally got one right!
